build.sh

```diff
@@ -71,7 +82,18 @@ else
 	popd
 fi
 
-if [ "$ARCH" = "arm" ]; then
+if [[ $TARGET_OS == 'android' ]]; then
+	echo "Setting gclient target_os to android"
+	echo "target_os = [ 'android' ]" >> $WEBRTC_DIR/.gclient
+	gclient config --name src $WEBRTC_REPO || exit 1
+	gclient sync --with_branch_heads -r $COMMIT || exit 1
+
+	echo "Installing Android build dependencies"
+	pushd $WEBRTC_SRC || exit 1
+	./build/install-build-deps-android.sh
+        gclient runhooks
```

Comment:
What does `gclient runhooks` do here?   Why is it necessary?

(Also, minor, adjust the spacing.)

Reply:
See above.  Also, one really stupid, annoying hard to solve issue was that `gclient sync` for Android requires you to manually accept the Play Services license agreement.  That's why I have `|| true` then `gclient runhooks`. https://bugs.chromium.org/p/webrtc/issues/detail?id=5578